### PR TITLE
add endpoint for all certificates

### PR DESCRIPTION
**Pull request by jw56578**

### CODE REVIEW

**Comment by kevincolten** [comment]
@morenol Please review, and merge/deploy when ready

File: lms/djangoapps/certificates/apis/v0/views.py [hunk 1]
```diff
@@ -7,14 +7,44 @@
...
-from lms.djangoapps.certificates.api import get_certificate_for_user
+from lms.djangoapps.certificates.api import get_certificate_for_user, get_all_certificates
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
 
 
 log = logging.getLogger(__name__)
 
+class CertificatesView(APIView):
```
Comment (morenol) [inline lms/djangoapps/certificates/apis/v0/views.py]: This is returning `NameError: name 'APIView' is not defined`. Please import that

File: lms/djangoapps/certificates/apis/v0/views.py [hunk 2]
```diff
@@ -7,14 +7,44 @@
...
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
```
Comment (morenol) [inline lms/djangoapps/certificates/apis/v0/views.py]: This authentication class does not exist in Ironwood, use this OAuth2AuthenticationAllowInactiveUser which is the same but was renamed to BearerAuthenticationAllowInactiveUser in Juniper

https://github.com/edx/edx-platform/commit/d7bd80a1006807a961b3dd5488f1dc58bbe1b87f

**Comment by morenol** [comment]
@kevincolten @jw56578  There are a couple of issues that I mentioned above, once the import is added and the BearerAuthenticationAllowInactiveUser is renamed it works fine on the devstack. Let me know when you fix that here and I will merge and deploy the changes

**Comment by jw56578** [comment]
@morenol @kevincolten 
Okay, I have made those changes

**Review by morenol — APPROVED** [review]

**Comment by morenol** [comment]
@kevincolten @jw56578 The deployment finished. This is on prod now.

**Comment by kevincolten** [comment]
@morenol @jw56578 Are we forgetting to import `six` somewhere?

Screen Shot 2020-08-17 at 6 49 34 PM

**Comment by morenol** [comment]
ohh, yes. We should import six in that file

**Comment by kevincolten** [comment]
@morenol Can you get that in?

**Comment by morenol** [comment]
Done in https://github.com/Zollege/edx-platform/pull/8

**Comment by morenol** [comment]
Could you check that again? @kevincolten

**Comment by kevincolten** [comment]
Working for me now! Thanks @jw56578 @morenol!

**Pull request merged**